docs/sources/developers/http_api/access_control.md

```diff
@@ -311,14 +311,80 @@ Content-Type: application/json; charset=UTF-8
 }
 ```
 
+#### Create role validation errors
+
+Permission validation only takes place when permission validation is enabled (`rbac.permission_validation_enabled = true`).
+
+> It has been enabled by default since Grafana 10.2
+
+##### Invalid action
+
+```http
+POST /api/access-control/roles HTTP/1.1
+Content-Type: application/json
+{
+	"Name": "Read Service Account with id 6",
+	"Permissions": [
+			{
+			"action": "serviceaccounts.permissions:reader",
+			"scope": "serviceaccounts:uid:6"
+		}
+	]
+}
+```
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+{
+	"extra": {
+		"validationError": "the provided action was not found in the list of valid actions: serviceaccounts.permissions:reader"
+	},
+	"message": "Permission contains an invalid action",
+	"messageId": "accesscontrol.permission-invalid-action",
+	"statusCode": 400,
+	"traceID": ""
+}
+```
+
+##### Invalid scope
+
```

Comment:
Please add a single introduction sentence here that describes what the reader is seeing in this section. Something like "The following example shows...".

Reply:
Done